addons/hr_timesheet/models/project.py

```diff
@@ -41,7 +41,7 @@ class Project(models.Model):
     timesheet_ids = fields.One2many('account.analytic.line', 'project_id', 'Associated Timesheets')
     timesheet_count = fields.Integer(compute="_compute_timesheet_count", groups='hr_timesheet.group_hr_timesheet_user')
     timesheet_encode_uom_id = fields.Many2one('uom.uom', related='company_id.timesheet_encode_uom_id')
-    total_timesheet_time = fields.Integer(
+    total_timesheet_time = fields.Float(
```

Comment:
We cannot do that in stable, it will alter the database schema

Reply:
> We cannot do that in stable, it will alter the database schema

The reason is that the field is non-stored.

Reply:
The field is recorded in ir.model.fields, so I don't think we can do that in stable.